paper-input-container.html

```diff
@@ -453,20 +455,21 @@
 
     _onBlur: function() {
       this._setFocused(false);
+      this._handleValue(this._inputElement, true);
     },
 
     _onInput: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
     _onValueChanged: function(event) {
-      this._handleValue(event.target);
+      this._handleValue(event.target, true);
     },
 
-    _handleValue: function(inputElement) {
+    _handleValue: function(inputElement, validateNow) {
```

Comment:
I have fallen into this trap many times before
http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html

What would you think about something like 

```
_handleValue: function(inputElement, options) {
...

options.bypassValidation && +      if (validateNow && 
```

?